src/src/WebServer/WebTemplateParser.cpp

```diff
@@ -293,12 +293,12 @@ void WebTemplateParser::getWebPageTemplateVar(const String& varName)
       if ((i == MENU_INDEX_RULES) && !Settings.UseRules) { // hide rules menu item
         continue;
       }
-#ifndef FEATURE_NOTIFIER
+#if FEATURE_NOTIFIER==0
```

Comment:
This could have been:
`#if !FEATURE_NOTIFIER`
but the end-result is the same 😃

Reply:
And that piece of code looks like it might still be wrong in the mega branch (#ifndef instead of #if !...)

Reply:
Tested that both ways and with and without Notifiers enabled, but it doesn't seem to make a difference, still works as intended (tab is shown or not, based on notifiers available or not), the code just looks a bit odd.
But you fixed it already so it will be corrected in due time 👍